packages/devtools-reps/src/object-inspector/utils/node.js

```diff
@@ -762,6 +771,69 @@ function getClosestNonBucketNode(item: Node) : Node | null {
   return getClosestNonBucketNode(parent);
 }
 
+function loadItemProperties(
```

Comment:
this function could probably be in its own module. It is sufficiently interesting in terms of dependencies and logic

```diff
  
+ const {
+   enumEntries,
+   enumIndexedProperties,
+   enumNonIndexedProperties,
+   getPrototype,
+   enumSymbols,
+ } = require("./client");
+ 

```

Reply:
what do you mean by own module ?

Reply:
it's own file

Reply:
sure